### PR TITLE
Prepare for 0.11.0.

### PR DESCRIPTION
**Pull request by non**
This adds support for 2.13.0-M1. It also uses Play-Json 2.6.0 for 2.12+,
which finally gets all our support projects using all the "stable"
cross versions of Scala we support.

Finally, it bumps the benchmarks to 2.12.

Thanks to Guillaume Massé for PR #73 which includes some of these changes.

### CODE REVIEW

**Comment by non** [comment]
This PR supersedes #73.

**Comment by non** [comment]
It would be nice to add some methods in Jawn that will go from `CharSequence` into `Long` and `Double`, since many ASTs (including Jawn's own) will need this kind of thing. I'm not sure whether we should do that here or in a future PR.

**Comment by non** [comment]
The +lines changed are a bit worrying, but on the other hand they are overwhelmingly tests and benchmarks, which seem reasonable.

File: parser/src/main/scala/jawn/Slice.scala [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+ *
+ *  - Slice.unsafe(s, start, limit) is for situations where the above
+ *    bounds-checking has already occurred. Only use this if you are
+ *    absolutely sure your arguments satisfy the above invariants.
+ *
+ * Slice's subSequence returns another slice. This means that when
+ * wrapping a very large string, garbage collection on the underlying
+ * string will not occur until all slices are freed.
+ */
+final class Slice private[jawn] (s: String, start: Int, limit: Int) extends CharSequence {
```
Comment (travisbrown) [inline parser/src/main/scala/jawn/Slice.scala]: Also I'm not sure it's necessary but I'm now paranoid about `Serializable`, and I could imagine that if circe ended up using these in its JSON string representation we'd want them to be `Serializable`.

Comment (travisbrown) [inline parser/src/main/scala/jawn/Slice.scala]: Right, that seems like a reasonable `equals` to me.

**Comment by non** [comment]
I think I am going to take @travisbrown's suggestion and move `Slice` and `Util` to a *jawn-util* module. Maybe tomorrow.

**Comment by non** [comment]
@travisbrown See what you think about the changes I just pushed. I'm not going to get this released until Sunday at the earlier, but apart from README updates I feel pretty good about where things are now.

**Comment by travisbrown** [comment]
@non Looks good to me! I think to fix the build you just need to add `util` to the `all` at the end so it'll get aggregated.

**Comment by non** [comment]
Just released 0.11.0. Should be available from Maven Central in a few hours.

**Comment by dwijnand** [comment]
@non Looking in Maven Central it doesn't look like any 2.13 release was made: http://repo1.maven.org/maven2/org/spire-math/ Am I missing something?

**Pull request merged**